modules/mobile/lib/scripts/appointments_list_validation.rb

```diff
@@ -34,6 +34,7 @@ def compare_sources
       web_only: in_web_only,
       mobile_only: in_mobile_only
     }
+    # robocop:disable Lint/Debugger
```

Comment:
I must not have done this `disable` right 🤔

Reply:
oh! 🤦 `rUbocop`!

Reply:
https://media.giphy.com/media/phyIS65F35Sco/giphy.gif